Cargo.toml

```diff
@@ -23,14 +23,14 @@ doctest = false
 bitflags = "2"
 bytes = "1"
 bytemuck = "1"
-chrono = "0.4, <= 0.4.31"                          # To meet crate MSRV target of Rust 1.60.
-image = { version = ">= 0.24.0", optional = true } # DynamicImage trait definitions changed between 0.23.14 and 0.24.0; we use trait from version 0.24.0 and later.
+chrono = "0.4"
 itertools = "0"
 log = "0"
 maybe-owned = "0"
 once_cell = "1"
 utf16string = "0"
 vecmath = "1"
+image = { version = "0.24", default-features = false, optional = true }
```

Comment:
1. `0.24    :=  >=0.24.0, <0.25.0`

2.  `image 0.24` requires msrv `1.63`, `image 0.25`  requires msrv `1.67.1`

```
$ cargo msrv
Fetching index
Determining the Minimum Supported Rust Version (MSRV) for toolchain aarch64-apple-darwin
Using check command cargo check

Check for toolchain '1.57.0-aarch64-apple-darwin' failed with:
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ error: failed to parse manifest at `/Users/fundon/dev/pdfium-render/Cargo.toml`                                                                │
│                                                                                                                                                │
│ Caused by:                                                                                                                                     │
│ namespaced features with the `dep:` prefix are only allowed on the nightly channel and requires the `-Z namespaced-features` flag on the       │
│ command-line                                                                                                                                   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
Check for toolchain '1.70.0-aarch64-apple-darwin' succeeded
Check for toolchain '1.63.0-aarch64-apple-darwin' succeeded

Check for toolchain '1.60.0-aarch64-apple-darwin' failed with:
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ error: package `image v0.24.9` cannot be built because it requires rustc 1.63.0 or newer, while the currently active rustc version is 1.60.0   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

Check for toolchain '1.61.0-aarch64-apple-darwin' failed with:
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ error: package `image v0.24.9` cannot be built because it requires rustc 1.63.0 or newer, while the currently active rustc version is 1.61.0   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

Check for toolchain '1.62.1-aarch64-apple-darwin' failed with:
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ error: package `image v0.24.9` cannot be built because it requires rustc 1.63.0 or newer, while the currently active rustc version is 1.62.1   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```

```
$ cargo msrv
Fetching index
Determining the Minimum Supported Rust Version (MSRV) for toolchain aarch64-apple-darwin
Using check command cargo check

Check for toolchain '1.57.0-aarch64-apple-darwin' failed with:
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ error: failed to parse manifest at `/Users/fundon/dev/pdfium-render/Cargo.toml`                                                                │
│                                                                                                                                                │
│ Caused by:                                                                                                                                     │
│ namespaced features with the `dep:` prefix are only allowed on the nightly channel and requires the `-Z namespaced-features` flag on the       │
│ command-line                                                                                                                                   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
Check for toolchain '1.70.0-aarch64-apple-darwin' succeeded

Check for toolchain '1.63.0-aarch64-apple-darwin' failed with:
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ error: package `image v0.25.4` cannot be built because it requires rustc 1.67.1 or newer, while the currently active rustc version is 1.63.0   │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
Unable to install toolchain with `rustup install 1.66.1-aarch64-apple-darwin`.
```

3. some default features of image are not used, so they can be turned off